package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "quickcredit-v2",
+    "version": "1.0.0",
+    "description": "quick credit api version 2 using postgresql",
+    "main": "src/index.js",
+    "scripts": {
+      "build": "babel src -d dist --source-maps",
+      "serve": "node dist/index.js",
+      "start": "cross-env ./node_modules/.bin/babel-node src/index.js",
+      "start:dev": "cross-env ./node_modules/.bin/nodemon src/index.js --exec babel-node",
+      "eslint": "eslint src/**/*.js --ignore-pattern \"node_modules/\""
+    },
+    "engines": {
+      "node": ">=8"
+    },
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/elemanhillary/QuickCreditV2.git"
+    },
+    "keywords": [
+      "QuickCredit",
+      "Version",
+      "2",
+      "PostgreSQL"
+    ],
+    "author": "eleman hillary barnabas",
+    "license": "ISC",
+    "bugs": {
+      "url": "https://github.com/elemanhillary/QuickCreditV2/issues"
+    },
+    "homepage": "https://github.com/elemanhillary/QuickCreditV2#readme",
+    "dependencies": {
```

Comment:
babel, eslint, prettier and jest are devDepencies, not dependencies. Research on the difference between devDepencies and dependencies and structure your packages accordingly.